uint8 sender_sid

Short[] questions
